### ISSUE TITLE
Alternative Control Interface for Non-Touch Devices

### ISSUE THREAD

**Issue by ff1992ff** [SkyTubeTeam/SkyTube#1030]
Hi everybody! I would like to request a feature. I primarily use Skytube on my smartphone, but I really also want to use it on my Fire TV stick. It runs Fire OS (which is basically an Android fork) and it can be installed by sideloading without any trouble. However, after starting the app, the trouble starts.

As the app was programmed with a touch control interface in mind, it is hardly to control with the remote control and its few function buttons:
- D-Pad
- Select
- Back
- ≡ : 3-parallel-lines-menu-button; I don't know the naming convention for this button 

It can be barely controlled, but it is not optimized for remote control and not a good user experience. I would really appreciate, if you could create an alternative control interface for non-touch input devices with remote control input like the Fire TV stick.

Skytube is the best YouTube client there is, but on my TV, I unfortunately have to use the bad stuff that Google has created. Thank you for your consideration and for the great work you have put out over the past years!

Kind regards,
Frank

**Comment by Dwedit** [comment]
Not a dev, just someone who also wants this feature.  
My idea here would be to pick either a button or the main window using the arrow keys, then center click the main window to start using that.  Then hit Back to return to the Buttons or Main Window selection.  Probably not a great design though.

**Comment by drogga** [comment]
@Dwedit, I don't think that someone will ever work on this, so here's a alternative for TV (it's not ideal \ perfect or flawless, but still...): https://github.com/yuliskov/SmartTubeNext
